### DISCUSSION TITLE
ema 50

### DISCUSSION THREAD

**Discussion by duvvurirao64** [pranjal-joshi/Screeni-py#146] [Ideas]
On my observation i found price crossing up ema 50 on 5min time frame are good for buy 
price crossing down ema 50 on 5 min time frame are good for selling intra day 
 two charts attached here with 
![vedanta-09122022](https://user-images.githubusercontent.com/120102901/206887403-d243547a-a6a8-48f8-8b4a-2f959cc64ab4.jpg)
![godrej091222022](https://user-images.githubusercontent.com/120102901/206887414-9b315cdf-a71c-47b1-92bb-17dfdf26e117.jpg)

request you to add these conditions in the code

**Comment by pranjal-joshi** [comment]
Hi @duvvurirao64

You can try editing the `screenipy.ini` file for your requirements.

My suggestion is to try
```
...
period = 10d
duration = 5m
useema = y
...
```
And then use `Option > 6 > Reversal at MA and give 50 as input to MA length`

The true power of `screenipy.ini` is you can personalize this for the timeframe and your trading style.

Hope this helps.

**Reply by duvvurirao64** [reply]
thanks sir, i will check it on monday live market
